src/components/Projects/ProjectDisplayer/ProjectDisplayer.js

```diff
@@ -35,29 +35,17 @@ const ProjectDisplayer = ({ org, repo }) => {
   );
   const { display } = useConfig();
 
-  if (loading) {
-    return (
-      <ProjectDisplayerContainer>
-        <Loader />
-      </ProjectDisplayerContainer>
-    );
-  }
-
-  if (repoInfoErr) {
-    const errorMessage = `An error has occurred while loading the ${repo} Github project. Please try again later.`;
-
-    return <ErrorContainer>{errorMessage}</ErrorContainer>;
-  }
+  function renderLanguages() {
+    if (languagesErr || !languages) {
```

Comment:
Re https://github.com/c-hive/dotdev/pull/130#issuecomment-623947932

Then I just wonder if it's possible to have a falsey `languages` without a `languagesErr`

Reply:
Eh, good point, I don't think so. I'd expect an error if `languages` was a falsy value.

https://github.com/c-hive/dotdev/pull/130/commits/7adfaaff53b3fba453ba544e8fb5df6f0d7a01a5